apps/web/app/api/auth/reset-password/route.ts

```diff
@@ -13,11 +15,10 @@ const passwordResetRequestSchema = z.object({
   requestId: z.string(), // format doesn't matter.
 });
 
-export default async function handler(req: NextApiRequest, res: NextApiResponse) {
-  // Bad Method when not POST
-  if (req.method !== "POST") return res.status(405).end();
```

Comment:
this is redundant now because we only define POST handler in this api route
<img width="416" alt="Screenshot 2025-02-24 at 1 47 35 PM" src="https://github.com/user-attachments/assets/44ea7ec7-f193-4c1d-b647-774508db662c" />